.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+---
+name: ci
+
+"on":
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  delivery:
+    runs-on: ubuntu-latest
```

Comment:
Looking at https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners 
> Note: The Ubuntu 20.04 virtual environment is currently provided as a preview only. The `ubuntu-latest` YAML workflow label still uses the Ubuntu 18.04 virtual environment.

Should we explicitly use newer?

Reply:
I have no preference for now.